### ISSUE TITLE
None of the tabs are describable in VoiceOver

### ISSUE THREAD

**Issue by astralbodies** [wordpress-mobile/WordPress-iOS#5078]
### Expected behavior
Each tab should be labeled with "Me", "Reader", etc when being described with VoiceOver.

### Actual behavior
Since we removed the visual labels I believe this broke VoiceOver.

### Steps to reproduce the behavior
Turn VoiceOver on and try scrolling through the tabs.
